packages/cli/lib/prompts/localDevUploadPrompt.js

```diff
@@ -0,0 +1,49 @@
+const readline = require('readline');
+const chalk = require('chalk');
+// const { promptUser } = require('./promptUtils');
+const { i18n } = require('../lang');
+
+const i18nKey = 'lib.prompts.localDevUploadPrompt';
+
+const makeUploadPrompt = message => {
+  return () => {
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    });
+
+    const formattedMessage = `${chalk.green('?')} ${chalk.bold(
+      message
+    )} ${chalk.dim('(Y/n)')} `;
+
+    let promiseResolve;
+
+    const promptPromise = new Promise(resolve => {
+      promiseResolve = resolve;
+      rl.question(formattedMessage, answer => {
```

Comment:
I tried a bunch of different things, but could not figure out a way to cancel prompts with our current version of `inquirer`. So for this one, just doing it manually with readline